.circleci/config.yml

```diff
@@ -82,68 +64,88 @@ jobs:
             DISPLAY: :99
   # Other tests, unrelated to typical code tests.
   Linting:
-    docker: [{ image: "circleci/node:8" }]
+    executor: { name: oss/node }
```

Comment:
These now use a newer version of `node`.  The "default" for our orbs is Node.js 10.